src/views/AbstractView.js

```diff
@@ -0,0 +1,123 @@
+import {Helper} from "../Helper";
+import {TitleCard} from "../cards/TitleCard";
+
+/**
+ * Abstract View Class.
+ *
+ * To create a new view, extend the new class with this one.
```

Comment:
Is this class actually extending the class? On another note, the use of `extend` helps to separate into different files, correct?

Reply:
It's the other way around.
`AbstractView` is a parent class (a base class), other classes can extend it by using the `extends` keyword at the class definition line.
E.g. `class HomeView extends AbstractView {}` => Child class `HomeView` is extending parent class `abstractView`.

Using a parent class, we can make sure, all required methods (functions) and properties exist in a child class.
A child class...
- Inherits all methods and properties of the parent class.
- Can override all public/protected methods and properties of the parent class.
- Can have additional methods and properties.

The parent class, in this case...
Is called abstract because we can't execute the class properly on it's own. We need different additional code for it. 
The additional code is defined in the child classes.
Since abstract classes aren't really implemented in JavaScript, it's simulated by throwing an error when you try to instantiate it with the `new` keyword.

E.g. 
```js
// Next line will throw an error because of line 39 in the AbstractView class.
foo = new AbstractView // call constructor method of the class.
```

In short:
An Abstract Class is like a partially built class. It is much like a document with blanks to fill in. It might be using English, but that isn't as important as the fact that some of the document is already written.
An abstract class is the foundation for another object. When a class says "I extend abstract class Y", it is saying "I use some methods or properties already defined in this other class named Y".